charts/coturn/values.yaml

```diff
@@ -0,0 +1,26 @@
+# The amount of coturn instances to run.  NOTE: Only one coturn can run per node due
+# to `hostNetwork`. If this number is higher than the amount of nodes that can
+# be used for scheduling (Also see `nodeSelector`) pods will remain in a
+# pending state untill you add more capacity.
+replicaCount: 1
+
+image:
+  repository: quay.io/wire/coturn
+  pullPolicy: IfNotPresent
+  # overwrite the tag here, otherwise `appVersion` of the chart will be used
+  tag: ""
+
+# If you have multiple deployments of coturn running in one cluster, it is
+# important that they run on disjoint sets of nodes, you can use nodeSelector to enforce this
+nodeSelector: {}
+
+podSecurityContext:
+  fsGroup: 31338
+
+securityContext:
+  # Pick a high number that is unlikely to conflict with the host
+  # https://kubesec.io/basics/containers-securitycontext-runasuser/
+  runAsUser: 31338
+
+coturnTurnListenPort: 3478
+coturnMetricsListenPort: 9641
```

Comment:
I would highly recommend to add the minimum amount of resources in terms of CPU and MEM usage. 
```
resources:
  requests:
    cpu: "300m"
    memory: "500Mi"
  limits:
    cpu: "600m"
    memory: "1Gi"
```

Reply:
I don't really have any good ideas about sensible resource request sizes and limits to set for this service, unfortunately.